lib/system/ansi_c.nim

```diff
@@ -111,22 +111,27 @@ type c_sighandler_t = proc (a: cint) {.noconv.}
 proc c_signal(sign: cint, handler: proc (a: cint) {.noconv.}): c_sighandler_t {.
   importc: "signal", header: "<signal.h>", discardable.}
 
-proc c_fprintf(f: File, frmt: cstring): cint {.
+type
+  CFile {.importc: "FILE", header: "<stdio.h>",
+          incompletestruct.} = object
+  CFileStar* = ptr CFile ## The type representing a file handle.
```

Comment:
why not `s/CFileStar/CFilePtr` ? seems more readable and it follows the existing nim convention

Reply:
You're right, will change it.